.code-samples.meilisearch.yaml

```diff
@@ -421,3 +428,63 @@ get_dump_status_1: |-
 phrase_search_1: |-
   curl -X POST 'http://localhost:7700/indexes/movies/search' \
   --data '{ "q": "\"african american\" horror" }'
+sorting_guide_update_sortable_attributes_1: |-
+  curl \
+  -X POST 'http://localhost:7700/indexes/books/settings' \
+  --data '{
+      "sortableAttributes": [
+          "author",
+          "price"
+      ]
+  }'
+sorting_guide_update_ranking_rules_1: |-
+  curl \
+  -X POST 'http://localhost:7700/indexes/books/settings/ranking-rules' \
+  --data '[
+    "words",
+    "sort",
+    "typo",
+    "proximity",
+    "attribute",
+    "exactness"
+  ]'
```

Comment:
Some times you use the subroute of a setting and sometime the main setting route. Is its willingly? Its not important I'm just curious ^^

Reply:
At the time I thought it may be less confusing for the user to only have on route. But I don't realise if it creates complexity or if it reduce it. So it's really up to your guts :')

Reply:
After thinking a bit more about it, I think it actually creates complexity and takes attention away from what's important in the guide.

I will use the direct subroutes.